webplugin/js/app/mck-sidebox-1.0.js

```diff
@@ -6445,6 +6451,11 @@ const firstVisibleMsg = {
                     data: w.JSON.stringify(messagePxy),
                     contentType: 'application/json',
                     success: function (data) {
+                        const {groupId} = messagePxy;
+
+                        kmLocalStorage.setItemToLocalStorage(
+                            groupId, { isCSATmodelClosed :false }
+                        )
```

Comment:
If user opens the old group then it will be break; 
try to add this real-time message